daemon/container.go

```diff
@@ -77,6 +77,20 @@ type CommonContainer struct {
 	logCopier *logger.Copier
 }
 
+// newBaseContainer creates a new container with its
+// basic configuration.
+func newBaseContainer(id, root string) *Container {
+	return &Container{
+		CommonContainer: CommonContainer{
+			ID:           id,
+			State:        NewState(),
+			execCommands: newExecStore(),
+			root:         root,
+			MountPoints:  make(map[string]*volume.MountPoint),
```

Comment:
Looks good except possibly for this line - it wasn't initialized previously on Windows.